consensus/src/checkpoint.rs

```diff
@@ -44,10 +44,14 @@ pub fn init_checkpoints(genesis: &mut ProtocolState) -> Result<(), ConsensusErrT
     Ok(())
 }
 
+/// A fork is considered short-range if either
+/// the fork point of the candidate chains are in the same epoch
+/// or the fork point is in the previous epoch with the same lock_checkpoint
```

Comment:
can the fork point be anywhere in the previous epoch? i thought it was only after the lock_checkpoint

Reply:
See [spec](https://github.com/MinaProtocol/mina/blob/32f529d8d8d712a44ee75be66061ce08cbdc8924/docs/specs/consensus/README.md#53-decentralized-checkpointing)
![Screenshot (419)](https://user-images.githubusercontent.com/8589224/150036389-3833336a-fc2f-4be7-85ea-4c9a6007b16f.png)

Reply:
From the spec, I only know the 2nd condition matches when the fork point is in the previous epoch with the same lock_checkpoint.